project_scrum/tests/test_project_scrum.py

```diff
@@ -0,0 +1,213 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import TransactionCase
+from odoo import fields, models, SUPERUSER_ID
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class TestProjectScrum(TransactionCase):
```

Comment:
without sensible assertions, most of those tests are pointless, that's just fake code coverage. In the create test, you can for example assert that it's the correct display name for meetings, correct counts for user stories, etc.
Generally, if you don't assert correct functioning of the code you call in a test, you can just as well leave out the test, because you just run code, don't test it